pkg/v1/cli/artifact/local.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package artifact
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli/common"
+)
+
+// LocalArtifact defines local artifact path
+type LocalArtifact struct {
+	Path string
```

Comment:
**nitpick**: It seems like we expect this to be an absolute path. Perhaps that's worth noting in the godoc?